src/utils/hooks/useHyperConvergeConfiguration.ts

```diff
@@ -7,6 +7,7 @@ import { K8sResourceCommon, useK8sWatchResource } from '@openshift-console/dynam
 
 export type HyperConverged = K8sResourceCommon & {
   spec: {
+    configuration?: { ksmConfiguration: { nodeLabelSelector?: object } };
```

Comment:
instead of object (as if i remember correctly can be a lot of things) and according the pr description this can be simply a {} ?

Reply:
So as I've expected, this was not good idea. Did you ever try to add {} there? If yes, then you must have seen the following:
```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.
```
:)